bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -50,13 +50,21 @@
 		<item-type>String</item-type>
 		<label>@text/channel-type.mielecloud.spinning_speed.label</label>
 		<description>@text/channel-type.mielecloud.spinning_speed.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Speed</tag>
```

Comment:
Even though the channel is called "speed", I'm wondering if this should be `Frequency`? It is reported in unit `ppm`, and for the Miele binding UoM is supported by `Number:Frequency`.

Reply:
Unit is rpm as this unit is also used on Miele washing machine datasheets. I'd go with frequency as mentioned in openhab documentation.